app/services/ApplicationStart.scala

```diff
@@ -0,0 +1,75 @@
+package services
+
+import com.google.inject.Singleton
+import io.prometheus.metrics.core.metrics.{Counter, Histogram}
+import io.prometheus.metrics.model.snapshots.Unit
+import utils.prometheus.CustomJvmMetrics
+
+import javax.inject.Inject
+
+@Singleton
+class ApplicationStart @Inject() (
+    // Add your dependencies here
+) {
```

Comment:
```suggestion
class ApplicationStart @Inject() () {
```

Not sure if what I suggest is the correct format.